src/frontend/src/components/RecordSearch/Record/DispositionQuestion.tsx

```diff
@@ -46,19 +88,26 @@ class DispositionQuestion extends React.Component<Props, State> {
     e.preventDefault();
     this.validateForm().then(() => {
       if (!this.state.missingFields && !this.state.invalidDate) {
-        store.dispatch(
-          answerDisposition(
-            this.props.case_number,
-            this.props.ambiguous_charge_id,
-            this.state.status,
-            this.state.conviction_date || "1/1/2020", // The date for a dismissal doesn't matter, but the backend expects something so here we are.
-            this.state.probation_revoked_date
-          )
-        )
+        this.setState<any>({
+          ["submitPending"] : false
+        });
+        this.dispatchAnswer();
```

Comment:
Should this not be pending?

Reply:
submitPending is true while waiting for the user to click Submit, so in this case the button becomes hidden immediately after the click and before dispatching the update to the backend.

Reply:
Maybe call it buttonClickPending then.